### PR TITLE
Integrated Service Discovery + Zoom Linking

### PR DESCRIPTION
**Pull request by anje123**
Integrated Service Discovery + Zoom Linking

### CODE REVIEW

File: server/api/routes/service.ts [hunk 1]
```diff
@@ -5,11 +5,12 @@ export class ServiceRoutes {
   public service: ServiceController = new ServiceController();
 
   public routes(app): void {
-    app.use(StrictlyAuthorizedRoutes);
```
Comment (flozender) [inline server/api/routes/service.ts]: Do we not need this anymore? @utkarsh-dixit I think you worked on this bit.

**Review by utkarsh-dixit — APPROVED** [review]
LGTM

File: server/api/routes/service.ts (hunk 1 above)
Comment (utkarsh-dixit) [inline server/api/routes/service.ts]: @flozender is right, we need this for making sure the requests are only available when the user is logged in. Any reasons why this was removed?

**Review by utkarsh-dixit — CHANGES_REQUESTED** [review]

File: server/api/routes/service.ts (hunk 1 above)
Comment (flozender) [reply server/api/routes/service.ts]: I think it was removed during a merge conflict, @anje123 can we get this back in?

**Review by utkarsh-dixit — APPROVED** [review]

**Pull request merged**